crates/nu-std/lib/log.nu

```diff
@@ -33,31 +36,47 @@ def now [] {
 
 # Log critical message
 export def "log critical" [message: string] {
-    if (current-log-level) > (CRITICAL_LEVEL) { return }
+    if (current-log-level) > (log CRITICAL_LEVEL) { return }
 
     print --stderr $"(ansi red_bold)CRT|(now)|($message)(ansi reset)"
 }
 # Log error message
 export def "log error" [message: string] {
-    if (current-log-level) > (ERROR_LEVEL) { return }
+    if (current-log-level) > (log ERROR_LEVEL) { return }
 
     print --stderr $"(ansi red)ERR|(now)|($message)(ansi reset)"
 }
 # Log warning message
 export def "log warning" [message: string] {
-    if (current-log-level) > (WARNING_LEVEL) { return }
+    if (current-log-level) > (log WARNING_LEVEL) { return }
 
     print --stderr $"(ansi yellow)WRN|(now)|($message)(ansi reset)"
 }
 # Log info message
 export def "log info" [message: string] {
-    if (current-log-level) > (INFO_LEVEL) { return }
+    if (current-log-level) > (log INFO_LEVEL) { return }
 
     print --stderr $"(ansi default)INF|(now)|($message)(ansi reset)"
 }
 # Log debug message
 export def "log debug" [message: string] {
-    if (current-log-level) > (DEBUG_LEVEL) { return }
+    if (current-log-level) > (log DEBUG_LEVEL) { return }
 
     print --stderr $"(ansi default_dimmed)DBG|(now)|($message)(ansi reset)"
 }
+
+# Log with custom message format and verbosity level
+# # Usage:
+# ```
+# use std
+# std log custom "my message" $"(ansi yellow)MY MESSAGE: %MSG%(ansi reset)" (std log WARNING_LEVEL)
+# ```
+export def "log custom" [
+    message: string, # Message inserted into the log template
+    format: string, # A string to be printed into stderr. Add %MSG% in place where the $message argument should be placed
+    log_level: int # Log's verbosity level
+    ] {
+    if (current-log-level) > ($log_level) { return }
+
+    print --stderr ($format | str replace "%MSG%" $message)
```

Comment:
i'd like to have `%LEVEL%` and `%DATE%` here, to be able to reconstruct the same format as the other `std log` commands :yum: